src/common/io.py

```diff
@@ -42,6 +43,15 @@
 
 SYS_LITTLE_ENDIAN = sys.byteorder == 'little'
 NCP_LARGE = 5000
+DISK_PROTECTION = 50*1024**2 #50mb
+
+#Get the available disk space at a given local path (minus 50mb to make sure we do not run out of space)
+def get_available_disk_space(local_path):
```

Comment:
```suggestion
def get_available_disk_space(local_path):
    """Get the available disk space at a given local path (minus 50mb to make sure we do not run out of space)."""
```
Might as well make this a docstring.